addons/mail/static/src/models/channel_member.js

```diff
@@ -0,0 +1,54 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+
+registerModel({
+    name: 'ChannelMember',
+    identifyingFields: ['channel', 'partner'],
+    recordMethods: {
+        /**
+         * Handles click on the avatar of the given member in the member list of
+         * this channel.
+         */
+        onClickAvatar() {
+            if (!this.partner) {
+                return;
+            }
+            this.partner.openChat();
+        },
+        /**
+         * Handles click on the name of the given member in the member list of
+         * this channel.
+         */
+        onClickName() {
+            if (!this.partner) {
+                return;
+            }
+            this.partner.openProfile();
+        },
+        _computeId() {
+            return this.partner.id;
+        },
```

Comment:
Not a good compute, id should be the id of channel.partner in python, that's the whole point of introducing this model